packages/build/src/plugins_core/functions/index.js

```diff
@@ -63,6 +68,22 @@ const getZisiParameters = ({ buildDir, featureFlags, functionsConfig, functionsD
   return { basePath: buildDir, config, manifest, featureFlags: zisiFeatureFlags }
 }
 
+// TODO: is this the best place for this logic? should this live elsewhere, e.g. in its own plugin?
+const writeToScheduleFile = async (zisiResult) => {
+  const schedule = zisiResult
+    .filter(({ config }) => Boolean(config.schedule))
+    .map(({ name, config }) => ({ route: `.netlify/functions/${name}`, schedule: config.schedule }))
```

Comment:
Instead of using "route", we could also just put the name of the function here and rely on Bitballoon to complete that to a HTTP Route. I decided against that, because this allows for advanced usecases like triggering the same functions with different HTTP query parameters on different schedules.

Reply:
> Could we use the name and then extend it to also include query parameters further down the line, if we decide to allow that?

Yes, good idea! Keep it lean 👍